assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -3,7 +3,6 @@ import { adjustViewport } from '../../util/chart'
 import { createResourcesText } from './d3ViewsNarrative'
 import d3tip from 'd3-tip'
 import './createResourceAccessChart.css'
-import '@fortawesome/fontawesome-free'
```

Comment:
I don't think this does anything.

Reply:
Yeah, looks like the only use we have should be covered by IconLabel.